src/library_browser.js

```diff
@@ -593,16 +593,9 @@ var LibraryBrowser = {
           Browser.mouseMovementY = Browser.getMovementY(event);
         }
 
-        // check if SDL is available
-        if (typeof SDL != "undefined") {
-          Browser.mouseX = SDL.mouseX + Browser.mouseMovementX;
-          Browser.mouseY = SDL.mouseY + Browser.mouseMovementY;
```

Comment:
AFAICT this code currently sets `mouseX` and `mouseY` to NaN since `undefined + number => NaN`